storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -5225,7 +5241,9 @@ xtrabackup_stats_func(int argc, char **argv)
 	    "xtrabackup: Using %lld bytes for buffer pool (set by "
 	    "--use-memory parameter)\n", xtrabackup_use_memory);
 
-	if(innodb_init(false))
+	init_mysql_environment();
```

Comment:
Please move `init_mysql_environment` before `xb_keyring_init_for_stats`. This way `--stats` will also work on encrypted tables :)